esmvalcore/preprocessor/_derive/siextent.py

```diff
@@ -48,16 +49,19 @@ def calculate(cubes):
             sic = cubes.extract_cube(Constraint(name="sic"))
         except iris.exceptions.ConstraintMismatchError:
             try:
-                sic = cubes.extract_cube(Constraint(name="siconca"))
+                sic = cubes.extract_cube(Constraint(name="siconc"))
             except iris.exceptions.ConstraintMismatchError as exc:
                 raise RecipeError(
                     "Derivation of siextent failed due to missing variables "
-                    "sic and siconca."
+                    "sic and siconc."
                 ) from exc
 
         ones = da.ones_like(sic)
         siextent_data = da.ma.masked_where(sic.lazy_data() < 15.0, ones)
         siextent = sic.copy(siextent_data)
-        siextent.units = "m2"
+        siextent.units = "1"  # unit is 1 as this is just a mask
```

Comment:
Thanks for the corrections @axel-lauer ! I think back in the day `area_statistics` was not handling well the units, so I had to set them directly in the derivation script. In any case looks like it has been fixed now. Maybe this comment could be written above the line to avoid the codacy issues about identation.

Reply:
Thanks! I moved the comments above the statement: https://github.com/ESMValGroup/ESMValCore/pull/2648/commits/7a1be2fa3e81659e7e40fa7b3eee75fd9d8583bd